regulations/static/regulations/js/source/views/main/main-view.js

```diff
@@ -249,6 +251,11 @@ var MainView = Backbone.View.extend({
 
     setHandlers: function() {
         this.applyTablePlugin();
+        this.clipboardLinks = new Clipboard('.clipboard-link', {
+            target: function(trigger) {
+                return trigger.nextElementSibling;
+            }
+        });
```

Comment:
Ideally this would
1. Insert the link into the DOM (rather than requiring it in the markup)
2. Fail gracefully if the browser doesn't support this. Gracefully here means no link
3. Scoping the query. I think there's a this.$el or similar.